content/docs/hooks-state.md

```diff
@@ -120,7 +120,9 @@ class Example extends React.Component {
   }
 ```
 
+<!-- textlint-disable -->
```

Comment:
This part needs to be deleted.

Reply:
Hi @hg-pyun,
Without this line, I cannot commit because of the `textlint` pre-commit hook.
Is there any other way to bypass `textlint` error message?

Reply:
React 공식문서 한국어 번역시 사용하려고 제가 작성한 textlint 기반 코드인데 bold(`**`)와 같이 쓰일 때 버그가 있는 거 같네요. 코멘트는 제가 따로 정리했습니다. (6cfddd3)